cartographer/mapping/trajectory_node_test.cc

```diff
@@ -36,7 +36,7 @@ TEST(TrajectoryNodeTest, ToAndFromProto) {
       sensor::CompressedPointCloud({{2.f, 3.f, 4.f}}).Decompress(),
       sensor::CompressedPointCloud({{-1.f, 2.f, 0.f}}).Decompress(),
       Eigen::VectorXf::Unit(20, 4),
-  };
+      transform::Rigid3d({1, 2, 3}, Eigen::Quaterniond(4., 5., -6., -7.))};
```

Comment:
nits: `1., 2., 3.`? Also for realism of the test: Use a unit quaternion, maybe simply calling `.normalized()`?